h2/src/test/org/h2/samples/SQLServerSample.java

```diff
@@ -0,0 +1,48 @@
+package org.h2.samples;
+
+import org.h2.tools.DeleteDbFiles;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class SQLServerSample {
```

Comment:
What is the purpose of this sample? It looks like something useless.

If you want to preserve it, please, add a copyright header (copy it from some other source file).

Reply:
You're right - with unit tests in place this is meaningless. I've removed it.